fenix/app/src/main/res/raw/initial_experiments.json

```diff
@@ -122,6 +171,76 @@
       "referenceBranch": "control",
       "featureValidationOptOut": false,
       "localizations": null
+    },
+    {
+      "schemaVersion": "1.12.0",
+      "slug": "testing-different-imagry-in-the-onboarding-tour",
+      "id": "testing-different-imagry-in-the-onboarding-tour",
+      "arguments": {},
+      "application": "org.mozilla.firefox_beta",
+      "appName": "fenix",
+      "appId": "org.mozilla.firefox_beta",
+      "channel": "beta",
+      "userFacingName": "Testing different imagry in the onboarding tour",
+      "userFacingDescription": "This experiment tests different sets of images for the same onboarding card messages to see if one is preferred.",
+      "isEnrollmentPaused": false,
+      "isRollout": false,
+      "bucketConfig": {
+        "randomizationUnit": "nimbus_id",
+        "namespace": "fenix-juno-onboarding-beta-2",
+        "start": 0,
+        "count": 10000,
+        "total": 10000
+      },
+      "featureIds": [
+        "juno-onboarding"
+      ],
+      "probeSets": [],
+      "outcomes": [],
+      "branches": [
+        {
+          "slug": "control",
+          "ratio": 1,
+          "feature": {
+            "featureId": "juno-onboarding",
+            "enabled": true,
+            "value": {
+              "enabled": true
+            }
+          }
+        },
+        {
+          "slug": "treatment-a",
+          "ratio": 1,
+          "feature": {
+            "featureId": "juno-onboarding",
+            "enabled": true,
+            "value": {
+              "enabled": true,
+              "cards": {
+                "default-browser": {
+                  "image-res": "onboarding_default_browser"
+                },
+                "sync-sign-in": {
+                  "image-res": "onboarding_sync"
+                },
+                "notification-permission": {
+                  "image-res": "onboarding_notification"
+                }
+              }
+            }
+          }
+        }
+      ],
+      "targeting": "((is_already_enrolled) || ((isFirstRun == 'true') && (app_version|versionCompare('114.!') >= 0) && (language in ['en'])))",
```

Comment:
should `is_already_enrolled` be `!is_already_enrolled` ?

Reply:
Nope, that's correct for sticky targeting!